### DISCUSSION TITLE
Pricing: are the first 2 credits free?

### DISCUSSION THREAD

**Discussion by nbcchen** [serverless/serverless#12757] [Q&A] [answered]
Does "2 Credits: FREE" mean the first 2 credits are free?
In other words, if I need 52 credits, do I pay for the 50 credits plan or the 300 credits plan?
https://www.serverless.com/pricing

**Comment by garethmcc** [comment, marked as answer]
@nbcchen The free tier provides 2 free credits. Once you subscribe then you do not have the 2 free credits. 

In addition, in your example, if you need 52 credits, you can also buy 1x50 and 1x15 credit bundles and not need to purchase 1x300.
